src/main/java/bio/terra/app/configuration/SameSiteCookieFilter.java

```diff
@@ -0,0 +1,91 @@
+package bio.terra.app.configuration;
+
+import jakarta.servlet.Filter;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.FilterConfig;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.servlet.http.HttpServletResponseWrapper;
+import java.io.IOException;
+import java.util.Collection;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+/**
+ * Servlet filter to ensure all cookies have SameSite=Lax attribute set. This addresses security
+ * vulnerability where JSESSIONID cookie lacks SameSite attribute.
+ */
```

Comment:
As you commented, this will set the `SameSite` value for ALL cookies, not just JSESSIONID. Could this have negative implications for any other cookies?